src/sentry/static/sentry/app/components/tagDistributionMeter.jsx

```diff
@@ -183,6 +184,7 @@ const Title = styled('div', {shouldForwardProp: isPropValid})`
 const TitleType = styled('div')`
   color: ${p => p.theme.gray4};
   font-weight: bold;
+  ${overflowEllipsis};
```

Comment:
Should this take up all available space? i.e. do we want it to have `flex: 1`? And should `Label` have a max-width instead of a fixed width? Also should `Label` be re-using `overflowEllipsis`?

Reply:
Good point - made the update to label and this is what they look like: 
![Screen Shot 2020-01-29 at 1 46 12 PM](https://user-images.githubusercontent.com/4830259/73400441-fcfd9f80-429d-11ea-8bce-33cb3b85b3b4.png)
![Screen Shot 2020-01-29 at 1 46 27 PM](https://user-images.githubusercontent.com/4830259/73400446-fff89000-429d-11ea-82c4-e830063bd560.png)
![Screen Shot 2020-01-29 at 1 46 50 PM](https://user-images.githubusercontent.com/4830259/73400449-01c25380-429e-11ea-94e0-1375f29d4e4c.png)

Reply:
Note: I didn't really want the label details to reach into the main type section.